### ISSUE TITLE
echo output missing on startup

### ISSUE THREAD

**Issue by gwww** [EdenEast/nightfox.nvim#98]
## Description

Remember when I said:
> The only thing that appear a bit odd in that I can't explain it is that without the setup options I don't see the "Ready!" message. With my setup options I do see the "Ready!" message.

when working on https://github.com/EdenEast/nightfox.nvim/issues/85#issuecomment-1071148648.

Well that is hanging around and something to do with Nightfox. Something in the plugin, which I have not been able to find, is causing the output of an `echo` to disappear. I can see the output there for a nanosecond if I watch real close, but it disappears.

I have narrowed it down to when I invoke the `vim.cmd 'colorscheme nightfox` command.

If I remove that command the echo output shows up and stays around. As soon as I add that command back the echo output flashes by. It is like something is injecting a `<CR>` into the startup stream.

Side note: I could not figure out how `vim.cmd 'colorscheme nightfox` loads the compiled nightfox theme... be happy to understand the lines of code that does that or how that works.

## Behavior

**Expected behavior**
I'd expect to not have output not be cleared.

## Configuration

Relevant nightfox's configuration...

**Reproduce**

To reproduce I have a `echo TEST!` right after my `colorscheme nightfox` command. Output "flashes" by when using nightfox colorscheme. Output stays there when using another (non nightfox) colorscheme.

This happens (as I mentioned in link above) when using the minimal init.lua.

## Additional info / context

Using neovim 0.6.1; wezterm latest.

**Comment by gwww** [comment]
I have found a super simple way to reproduce.

Using `minimal_init.lua` start nvim as instructed in minimal. Notice the "Ready!" message is not visible.

Now, comment out line 36 in `minimal_init.lua` where the `colorscheme` is set to nightfox. Start nvim again using the minimal instructions. Notice the "Ready!" message is visible.

The reason I've created a bug was I could not figure out where a debug print "disappeared" to when I was working on something else.

**Comment by gwww** [comment]
I have narrowed it down to a single line! 

`highlight MsgArea guifg=#aeafb0 guibg=NONE gui=NONE guisp=NONE`

When the above line is in the theme, the message area "flashes" by such that the message is missed. When the line is not in the theme everything works as expected.

Now the real question is what to do now? This is not an obvious NightFox theme problem.

I looked at a couple of other color schemes and they don't set MsgArea hilighting. 

vim (as opposed to neovim) doesn't have this problem as it's startup is different than nvim in that messages are printed right away before going into full screen mode. Also, vim seems to ignore the MsgArea setting. So this is unique to nvim.

nvim does recognize the setting but seems to clear the MsgArea when setting the hilight.

nvim bug report?? Workaround to not set MsgArea?
